README.md

```diff
@@ -273,6 +285,27 @@ currently only `8080`, but it may change in the future.)
 If your existing application doesn't allow you to configure port number it
 listens on, Cloud Run currently doesn't allow customizing the `PORT` value.
 
+### Can Cloud Run receive events?
```

Comment:
This section is bit too long and looks like docs (something we should try to avoid) but it’s cool for now.